src/config.js

```diff
@@ -87,6 +87,13 @@ export function applyConfigToArgv({
     }
 
     newArgv[option] = configObject[option];
+
+    const coerce = options[decamelizedOptName].coerce;
```

Comment:
It would be nice to cleanup this method further in a followup (e.g. `options[decamelizedOptName]` could be assigned to a variable near line 48 and then use it where we are currently using `options[decamelizedOptName]`, only to make the rest of this method a bit more readable).